src/app/tab/time/time.component.html

```diff
@@ -1,111 +1,289 @@
 <div
   class="bracket bracket--left"
   *ngIf="clock.brackets.enabled && clock.brackets.left !== ''"
-  [ngClass]="{'dim':clock.brackets.dim}"
+  [ngClass]="{ dim: clock.brackets.dim }"
   [style.fontSize]="shared.getFontSize(clock.brackets.scaling, 5)"
   [style.transform]="shared.getOffset(clock.brackets.offset)"
-  [style.marginRight]="getPadding(clock.brackets.padding)">
-  {{clock.brackets.left}}
+  [style.marginRight]="getPadding(clock.brackets.padding)"
+>
+  {{ clock.brackets.left }}
 </div>
 <div class="clockContent">
   <div class="time" *ngIf="!clock.analog.enabled">
     <div class="timeDigit hour">
-      <span [style.width]="getWidth(clock.padding)">{{getHour(clock.timezone, clock.twentyFour, clock.twoDigit, 1)}}</span>
-      <span [style.width]="getWidth(clock.padding)" *ngIf="getHour(clock.timezone, clock.twentyFour, clock.twoDigit, 2)">{{getHour(clock.timezone, clock.twentyFour, clock.twoDigit, 2)}}</span>
+      <span [style.width]="getWidth(clock.padding)">{{
+        getHour(clock.timezone, clock.twentyFour, clock.twoDigit, 1)
+      }}</span>
+      <span
+        [style.width]="getWidth(clock.padding)"
+        *ngIf="getHour(clock.timezone, clock.twentyFour, clock.twoDigit, 2)"
+        >{{
+          getHour(clock.timezone, clock.twentyFour, clock.twoDigit, 2)
+        }}</span
+      >
     </div>
     <div
       *ngIf="clock.delimiter.enabled"
       class="delimiter"
-      [ngClass]="{'dim':clock.delimiter.dim, 'blink':getBlink(clock.delimiter.blink)}"
+      [ngClass]="{
+        dim: clock.delimiter.dim,
+        blink: getBlink(clock.delimiter.blink)
```

Comment:
To follow convention, keep the quotes around the class names. Starting to wonder if this is a linter doing this though. I really need to adjust my linting settings, I currently don't use it in my local env lol.

Reply:
This was the way that prettier format the properties. I have fixed it by adding the "quote-props": "preserve" to Package.json so that it stays consistant.